homeassistant/components/tessie/sensor.py

```diff
@@ -219,9 +221,11 @@ class TessieSensorEntityDescription(SensorEntityDescription):
     ),
     TessieSensorEntityDescription(
```

Comment:
This needs an available function to filter out None values.

Reply:
Right, I wasnt sure if that was required. Ive reverted the commit removing it. I wll add this functionality everywhere in a seperate PR.